.cargo/audit.toml

```diff
@@ -12,6 +12,14 @@ ignore = [
   "RUSTSEC-2020-0168", # mach is unmaintained
   "RUSTSEC-2022-0061", # parity-wasm is deprecated
   "RUSTSEC-2024-0370", # proc-macro-error is unmaintained
+  # Instant is unmaintained, but there is a replacement in
+  # the pipeline - see
+  # https://github.com/libp2p/rust-libp2p/pull/5674
+  # We will likele need to get rid of `backoff` dependency
+  # in favour of something that is maintained too - see
+  # https://github.com/ihrwein/backoff/pull/70. Last commit
+  # to master was 3 years ago.
+  "RUSTSEC-2024-0384",
```

Comment:
And why do we still ignore this when it's been fixed? 

https://github.com/console-rs/indicatif/releases/tag/0.17.9
> Replace instant with web-time

Reply:
Ah I see, it's used by other dependencies